key/02c/01.py

```diff
@@ -1,3 +0,0 @@
-import ipywidgets as widgets
```

Comment:
Was this file supposed to be deleted?

Reply:
Yeah, I created it temporarily when I thought I needed to use it, but the problem is if you host an answer in a non-module file, you can't successfully export the cell to nbdev.  So I removed this.